### DISCUSSION TITLE
this project is dead?

### DISCUSSION THREAD

**Discussion by kikyoulg** [felixse/FluentTerminal#988] [General]
long time no update

**Comment by felixse** [comment]
Not dead yet.

The current architecture is basically a dead end in regards to most bug reports/feature requests.
I'm working on a rewrite with Windows App SDK in the winui3 branch right now, but it will take some more time.

**Reply by AnderssonPeter** [reply]
Does this mean we will loose the glass background? If i remember correctly Winui3 does not support it?

**Reply by microhobby** [reply]
some news?

**Reply by GitHub123cp** [reply]
> Not dead yet.
> 
> The current architecture is basically a dead end in regards to most bug reports/feature requests. I'm working on a rewrite with Windows App SDK in the winui3 branch right now, but it will take some more time.

Excuse me, are you still updating?

**Comment by ovizii** [comment]
Any further news on the progress of Fluent Terminal?

**Comment by belst-n** [comment]
Hi, is it dead yet?
